.github/workflows/CompatHelper.yml

```diff
@@ -0,0 +1,16 @@
+name: CompatHelper
+on:
+  schedule:
+    - cron: '00 00 * * *'
+  workflow_dispatch:
+jobs:
+  CompatHelper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pkg.add("CompatHelper")
+        run: julia -e 'using Pkg; Pkg.add("CompatHelper")'
+      - name: CompatHelper.main()
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          COMPATHELPER_PRIV: ${{ secrets.COMPATHELPER_PRIV }}  # optional
```

Comment:
Do we have to define `COMPATHELPER_PRIV` token or not? If so, could we use `DOCUMENTER_KEY` as in [here](https://github.com/PerezHz/TaylorIntegration.jl/blob/76cd51a807f27bd7a9a51fc96e4a088884f83248/.github/workflows/docs.yml#L27)?

Reply:
indeed, im no expert but i think you can also remove it 
 see other packages eg .https://github.com/SciML/DifferentialEquations.jl/blob/master/.github/workflows/CompatHelper.yml

Reply:
Can you comment it, so if we require it it is a simple change? Then I merge this!